.travis.yml

```diff
@@ -40,3 +41,12 @@ script:
     if [ "$TRAVIS_TAG" == "1.0-stable" ]; then
       docker run -it f5devcentral/containthedocs publish-product-docs-to-prod connectors/k8s-bigip-ctlr v1.0
     fi
+
+deploy:
+  - provider: script
+    skip_cleanup: true
+    on:
+      branch: 1.1.0
+      repo: F5Networks/k8s-bigip-ctlr
+    script:
+      - ./build-tools/deploy-docs.sh publish-product-docs-to-prod connectors/k8s-bigip-ctlr v1.1.1
```

Comment:
The version should be 'v1.1'. The product-docs script only accepts 'vX.Y[-beta]' or a single word comprised of alpha-numeric characters.